Yoast/wordpress-seo
# Added days to how to block

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Add an How-To block in gutenberg,
* Add a duration.
* Click `Preview` to see that the duration is formatted as intended.

## Quality assurance


Fixes #10700
